pkg/server/sandbox_status.go

```diff
@@ -200,3 +144,71 @@ func toCRISandboxStatus(meta sandboxstore.Metadata, state runtime.PodSandboxStat
 		Annotations: meta.Config.GetAnnotations(),
 	}
 }
+
+// TODO (mikebrow): discuss predefining constants structures for some or all of these field names in CRI
+type sandboxInfo struct {
+	Pid         uint32                    `json:"pid"`
+	Status      string                    `json:"processStatus"`
+	NetNS       string                    `json:"netNamespace"`
```

Comment:
already in the spec in the right spot

Reply:
This is critical information, it's quite hard to find in runtime spec. We want to highlight it.

We may need to remove runtime spec or higher its verbosity in the future. Do not want this to be affected.

Reply:
I'll keep `NetNS` here, because we have `NetNSClosed` just below it, which makes sense to me.

Reply:
`| grep` :-)

Reply:
Will also remove then.